frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -41,19 +41,32 @@ type SegmentInfo = {
 const exportStatisticsToCSV = (
   segmentInformation: Array<SegmentInfo>,
   tracingId: string,
-  groupId: number,
+  groupIdToExport: number,
 ) => {
-  if (segmentInformation.length < 0) {
+  if (segmentInformation.length === 0) {
     return;
   }
   const segmentStatisticsAsString = segmentInformation
     .map(
-      (segmentInfo) =>
-        `${segmentInfo.segmentId},${segmentInfo.segmentName},${segmentInfo.groupId},${segmentInfo.groupName},${segmentInfo.volumeInVoxel},${segmentInfo.volumeInNm3},${segmentInfo.boundingBoxTopLeft},${segmentInfo.boundingBoxPosition}`,
+      (row) =>
+        [
+          row.segmentId,
+          row.segmentName,
+          row.groupId,
+          row.groupName,
+          row.volumeInVoxel,
+          row.volumeInNm3,
+          row.boundingBoxTopLeft,
+          row.boundingBoxPosition,
+        ]
+          .map(String) // convert every value to String
+          .map((v) => v.replaceAll('"', '""')) // escape double quotes
+          .map((v) => (v.includes(",") || v.includes('"') ? `"${v}"` : v)) // quote it if necessary
+          .join(","), // comma-separated
     )
-    .join("\n");
+    .join("\r\n"); // rows starting on new lines
```

Comment:
```suggestion
    .join("\n"); // rows starting on new lines
```

I think, `\r` only interesting for windows users, but we haven't used that anywhere else yet. So, I wouldn't start now unless there is a strong reason for it.

Reply:
this was definitely by accident, maybe VS code added this or something, I'll delete it